subsystems/motor/ServoSubsystem.java

```diff
@@ -0,0 +1,242 @@
+	 * Can also modify their speed with a SpeedModifier for things like scaling or brownout protection.
+	 *
+	 * @param motors
+	 *        The SpeedControllers in this subsystem.
+	 *        Can be a single Servo or multiple Servo.
+	 */
+	public ServoSubsystem(Servo... servos) {
+		this("ServoSubsystem", servos);
+	}
+
+	@Override
+	protected void initDefaultCommand() {
+		setDefaultCommand(new Idle(this));
+	}
+
+	/**
+	 * Get the servo position.
+	 *
+	 * <p>
+	 * Servo positions range from 0.0 to 1.0 corresponding to the range of full left to full right.
+	 *
+	 * @return Position from 0.0 to 1.0.
+	 */
+	public double get() {
+		return lastPosition;
+	}
+
+	/**
+	 * Get the servo position.
+	 *
+	 * <p>
+	 * Servo positions range from 0.0 to 1.0 corresponding to the range of full left to full right.
+	 * (This method is equivalent to the {@link #get() get()} method.)
+	 *
+	 * @return Position from 0.0 to 1.0.
+	 */
+	public double getPosition() {
+		return get();
+	}
+
+	/**
+	 * Get the servo angle.
+	 *
+	 * @return The angle in degrees to which the servo is set.
+	 */
+	public double getAngle() {
+		return convertPositionToAngle(get());
+	}
+
+	/**
+	 * Set the servo position.
+	 *
+	 * <p>
+	 * Servo values range from 0.0 to 1.0 corresponding to the range of full left to full right.
+	 *
+	 * @param position
+	 *        Position from 0.0 to 1.0.
+	 */
+	public void set(double position) {
+		if (isInverted) {
+			position = invertPosition(lastPosition);
+		}
+		for (Servo servo : servos) {
+			servo.set(position);
+		}
+		lastPosition = position;
+	}
+
+	/**
+	 * Set the servo position.
+	 *
+	 * <p>
+	 * Servo positions range from 0.0 to 1.0 corresponding to the range of full left to full right.
+	 * (This method is equivalent to the {@link #set(double) set()} method.)
+	 *
+	 * @param value
+	 *        Position from 0.0 to 1.0.
+	 */
+	public void setPosition(double position) {
+		set(position);
+	}
+
+	/**
+	 * Set the servo angle.
+	 *
+	 * @return The angle in degrees to which the servo is set.
+	 */
+	public void setAngle(double degrees) {
+		if (isInverted) {
+			degrees = invertDegree(degrees);
+		}
+		for (Servo servo : servos) {
+			servo.setAngle(degrees);
+		}
+	}
+
+	/**
+	 * Get whether this entire servo is inverted.
+	 *
+	 * @return isInverted
+	 *         The state of inversion, true is inverted.
+	 */
+	public boolean getInverted() {
+		return isInverted;
+	}
+
+	/**
+	 * Set whether this entire servo is inverted.
+	 * Note that this will also convert the values returned by {@link #get() get()}, {@link #getPosition() getPosition()},
+	 * and {@link #getAngle() getAngle()} to the new coordinate system, so anything continuously reading these methods
+	 * will see a discontinuity.
+	 * 
+	 * @param isInverted
+	 *        The state of inversion, true is inverted.
+	 */
+	public void setInverted(boolean isInverted) {
+		if (isInverted != this.isInverted) {
```

Comment:
Also, bad practice to have a local variable with the same name as a field, and differentiating between the two via usage vs non usage of `this.`